examples/js/test-js.js

```diff
@@ -22,54 +22,74 @@ const cppCodeBlock = `#include <iostream>
 int main()
 {
   auto curried_add = [](int x) -> function<int(int)> { return [=](int y) { return x + y; }; };
-  
+
   auto answer = curried_add(7)(8);
   std::cout << answer << std::endl;
-  
+
   return 0;
 }`;
 
 const TestJs = () => (
-  <Deck>
-    <Slide slideNum={1}>
+  <Deck
+    template={({ numberOfSlides, slideNumber }) => (
+      <FlexBox
+        justifyContent="space-between"
+        position="absolute"
+        bottom={0}
+        width={1}
+      >
+        <Text fontSize={12}>
+          Slide {slideNumber + 1} of {numberOfSlides}
```

Comment:
If we want the slide # to match the url, then it should be 0 indexed. Might be confusing if they change the url manually

Reply:
Good point!